packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -16,32 +16,9 @@ import Collection from '../collection/redis'
 import parseDuration from '../_parseDuration'
 import patch from '../patch'
 import readChunk from '../_readStreamChunk'
+import { extractIpFromVmNetworks as extractIp } from '../_extractIpFromVmNetworks'
```

Comment:
Why not using the real name?

Reply:
For simplification, i'm not against using the real name.

Reply:
Either use the name of the module or simplify it.